lib/Sema/TypeCheckDecl.cpp

```diff
@@ -512,10 +512,13 @@ static void checkForEmptyOptionSet(const VarDecl *VD, TypeChecker &tc) {
   if (!PBD)
     return;
   
-  auto entry = PBD->getPatternEntryForVarDecl(VD);
-  
+  auto initIndex = PBD->getPatternEntryIndexForVarDecl(VD);
+  auto init = PBD->getInit(initIndex);
+  if (!init)
+    return;
+
   // Make sure property is being set with a constructor
-  auto ctor = dyn_cast<CallExpr>(entry.getInit());
+  auto ctor = dyn_cast<CallExpr>(init);
```

Comment:
Hm, why the indirection through the index?

Reply:
@CodaFi removed the direct accessors for requestification

Reply:
Gotcha. Not that it's important, but you could also compact this a little by using `dyn_cast_or_null`.

Reply:
Oh, that's useful! Didn't know about that. Done!